CHANGELOG.md

```diff
@@ -1,48 +1,62 @@
 # Unreleased
 
-Nothing Yet!
+This release contains a number of new features and several fixes.
 
 
-# Version 0.15.0 (unreleased)
+## Windows codesigning
+
+cargo-dist can automatically codesign Windows EXEs and MSIs using SSL.com's eSigner cloud signing service.
+
+Although there are many ways to do code signing, this process is specifically concerned with ensuring Windows SmartScreen recognizes the authenticity of the signatures and doesn't prevent your users from running the application. Otherwise, any user who downloads your application with a web browser will get a popup warning them against running it. (Alternative methods of downloading and installing, such as cargo-dist's powershell installers do not trigger SmartScreen.)
+
+* [docs](https://opensource.axo.dev/cargo-dist/book/signing-and-attestation.html#windows-artifact-signing-with-sslcom-certificates)
+* impl @Gankra [feat: reimplement ssldotcom-windows-sign](https://github.com/axodotdev/cargo-dist/pull/1036)
 
-This release contains a number of new features and several fixes.
 
 ## Improvements for installing tools within GitHub Actions
 
 Previously, our installers would update the user's `PATH` for local commandline access but wouldn't set themselves up in the `PATH` within GitHub Actions. We've added a feature to the shell and Windows installers which detects if they're running within Actions and adds the newly-installed tool to `GITHUB_PATH` for future tasks within the job.
 
 * impl @Gankra [feat: add paths to GITHUB_PATH if available](https://github.com/axodotdev/cargo-dist/pull/1047)
 
+
+## Custom installer output
+
+When a shell or powershell installer runs successfully it will print out "everything's installed!", but if you want a different message that better matches your app's look-and-feel, or provides a more useful call-to-action, you can now do that with [the `install-success-msg` setting](https://opensource.axo.dev/cargo-dist/book/reference/config.html#install-success-msg).
+
+* [docs](https://opensource.axo.dev/cargo-dist/book/reference/config.html#install-success-msg)
+* impl @ashleygwilliams [feat: custom success msg](https://github.com/axodotdev/cargo-dist/pull/1102)
+
+
 ## Always treat releases as the latest stable release
```

Comment:
this is maybe a little confusing- i feel like this reads like we made this the default behavior and we definitely did not